package.json

```diff
@@ -41,8 +41,9 @@
 	"scripts": {
 		"start": "react-scripts start",
 		"build": "react-scripts build",
-		"buildCI": "CI= react-scripts build",
-		"test": "react-scripts test",
+		"buildCI": "react-scripts build",
```

Comment:
`"buildCI": "CI= react-scripts build"` don't change this command or else Netlify fails to build if there are warnings in code.